tla-io/src/test/scala/at/forsyte/apalache/io/annotations/TestAnnotationParser.scala

```diff
@@ -110,6 +110,23 @@ class TestAnnotationParser extends AnyFunSuite with Checkers {
       .map(r => fail("Expected a failure. Found: " + r))
   }
 
+  test("parse variants in type aliases") {
+    val extractedText =
+      """MESSAGE =""" + "\n" + """  Req({ ask: Int })      """ + "\n" + """        | Ack({ success: Bool })"""
```

Comment:
Seem like this could be a lot clearer using a multi-line string, no?

Reply:
For sure. The problem is that `stripMargin` is using `|` as a delimiter. So the tests were breaking. I could not find a better solution :(

Reply:
Oh I see! I think we can do

```suggestion
      """#MESSAGE =
         #|  Req({ ask: Int })
         #|  Ack({ success: Bool })""".stripMargin('#')
```

Or using whatever char might be better in place of `#`.